plugins/BEdita/Core/src/Filesystem/Thumbnail/GlideGenerator.php

```diff
@@ -54,10 +56,15 @@ class GlideGenerator extends ThumbnailGenerator
      * @param \BEdita\Core\Model\Entity\Stream $stream Stream entity instance.
      * @param array $options Thumbnail options.
      * @return string
+     * @throws BadRequestException
```

Comment:
It's generally advisable to use fully-qualified class names in docblock annotations:

```suggestion
     * @throws \Cake\Http\Exception\BadRequestException
```